jvm/sargon-android/src/main/java/com/radixdlt/sargon/extensions/DappToWalletInteractionUnvalidated.kt

```diff
@@ -0,0 +1,14 @@
+package com.radixdlt.sargon.extensions
+
+import com.radixdlt.sargon.DappToWalletInteractionUnvalidated
+import com.radixdlt.sargon.dappToWalletInteractionUnvalidatedToJsonBytes
+import com.radixdlt.sargon.newDappToWalletInteractionUnvalidatedFromJsonBytes
+
+@Throws(SargonException::class)
+fun DappToWalletInteractionUnvalidated.Companion.fromJson(json: String) =
+    newDappToWalletInteractionUnvalidatedFromJsonBytes(json.toByteArray().toBagOfBytes())
+
+@Throws(SargonException::class)
+fun DappToWalletInteractionUnvalidated.toJson() =
```

Comment:
Does this method throw an exception @CyonAlexRDX ?

Reply:
No

Reply:
I think @jakub-rdx if you open the generated `sargon.kt` you will see that this method does not throw. Neither this one https://github.com/radixdlt/sargon/pull/155/files#r1625795145

Reply:
updated